cartridge/confapplier.lua

```diff
@@ -381,18 +382,11 @@ local function boot_instance(clusterwide_config)
         box_opts.replicaset_uuid = assert(replicaset_uuid)
         box_opts.replication_connect_quorum = 1
         box_opts.replication = topology.get_fullmesh_replication(
-            topology_cfg, box_opts.replicaset_uuid
-        )
-        -- Workaround for https://github.com/tarantool/tarantool/issues/3760
-        -- which wasn't cherry-picked to tarantool 1.10 yet.
-        -- Due to the bug box_opts.replication_connect_quorum was ignored
-        -- and box.cfg used to hang
-        table.remove(
-            box_opts.replication,
-            utils.table_find(
-                box_opts.replication,
-                pool.format_uri(vars.advertise_uri)
-            )
+            topology_cfg, replicaset_uuid,
+            -- Workaround for https://github.com/tarantool/tarantool/issues/3760
```

Comment:
Seems patch has been already cherry-picked https://github.com/tarantool/tarantool/issues/3760

Not sure we should support some workarounds.

Reply:
I've removed stale comment, but I'd like to preserve the workaround - it doesn't affect anything.